### DISCUSSION TITLE
How can I obtain the .lab file?

### DISCUSSION THREAD

**Discussion by Dislorid** [fishaudio/fish-speech#615] [Q&A]
When extracting semantics in batches,  .lab file is required. How can I get the .lab file? Do I need to use any specific software to generate it?

**Comment by tjiang4966** [comment]
The .lab annotation file only needs to contain the transcription of the audio, with no special formatting required. For example, if hi.mp3 says "Hello, goodbye," then the hi.lab file would contain a single line of text: "Hello, goodbye."

**Comment by lost723** [comment]
https://text-labeler.pages.dev/  generate on this site
